suite-common/formatters/src/kinds/prepareCryptoAmountFormatter.ts

```diff
@@ -0,0 +1,58 @@
+import BigNumber from 'bignumber.js';
+
+import { networksCompatibility as NETWORKS, NetworkSymbol } from '@suite-common/wallet-config';
+import {
+    localizeNumber,
+    networkAmountToSatoshi,
+    formatCoinBalance,
+} from '@suite-common/wallet-utils';
+import { PROTO } from '@trezor/connect';
+
+import { makeFormatter } from '../makeFormatter';
+import { FormatterConfig } from '../types';
+
+export type CryptoAmountFormatterInputValue = string | number | BigNumber;
+
+export type CryptoAmountFormatterDataContext = {
+    isBalance?: boolean;
+    symbol?: string;
+};
+
+export const prepareCryptoAmountFormatter = (config: FormatterConfig) =>
+    makeFormatter<CryptoAmountFormatterInputValue, string, never, CryptoAmountFormatterDataContext>(
+        (value, dataContext) => {
+            const { symbol, isBalance } = dataContext;
+            const { locale, bitcoinAmountUnit } = config;
+
+            const lowerCaseSymbol = symbol?.toLowerCase();
```

Comment:
Why not to use currencySymbolFormatter directly here instead of copying?

Reply:
The functionality is different.